include/zombie/tock/tree.hpp

```diff
@@ -70,6 +71,17 @@ struct TockTree {
       parent->children.erase(data.range.beg);
     }
 
+
+    void delete_leaf_children() {
+      for (auto it = children.begin(); it != children.end();) {
+        if (it->second.children.empty())
```

Comment:
can we test for zombie? this currently recursively remove microwave. removing them might increase cost when remat because you need to look for a larger microwave. let's do that another time.

Reply:
evicted microwave with no submicrowave will be empty, plus return value need not be in, because you can return a value created elsewhere.

Reply:
see https://github.com/MarisaKirisame/zombie/blob/main/test/zombie_test.cc#LL224C18-L224C29